src/index.html

```diff
@@ -11,7 +11,204 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="set">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="name">
+            Name:
+          </label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="HowOld">
+            How old are You?
+            <input
+              type="number"
+              name="HowOld"
+              id="HowOld"
+              value="12"
+              min="1"
+              max="100"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="FullDate">
+              Full date of birth:
+            <input
+              type="date"
+              name="FullDate"
+              id="FullDate"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="check">
+            I accept the term of agreement
+            <input
+              type="checkbox"
+              name="check"
+              id="check"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="set">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="mail">
+            E-mail:
+            <input
+              type="email"
+              name="mail"
+              id="mail"
+              placeholder="email@example.com"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="pass">
+            Password:
+            <input
+              type="password"
+              name="pass"
+              id="pass"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="set">
+        <legend>An interesting fact about you</legend>
+        <div class="form-field">
+            Do you love cats?
+          <label for="rad">
+            <input
+              type="radio"
+              name="rad"
+            > Yes
+          </label>
+          <label for="rad">
```

Comment:
Remove `for` attributes because now your labels doesn't work, to connect label and input field you should add id attrubute, but it's redundant it this case because your input in wrapped by label